widget/entry.go

```diff
@@ -83,6 +85,8 @@ func NewEntry() *Entry {
 }
 
 // NewEntryWithData returns an Entry widget connected to the specified data source.
+//
+// Since: 2.0.0
```

Comment:
Commenting here because I can't comment on unchanged lines.

Lines 99-101
```go
entry.OnChanged = func(s string) {
	data.Set(s)
}
```
Could be simplified to;
```go
entry.OnChanged = data.Set
```